NEWS.md

```diff
@@ -23,6 +23,8 @@ Language changes
    expression within a given `:toplevel` expression to make use of macros
    defined earlier in the same `:toplevel` expression. ([#53515])
 
+ - Infinite loops are no longer undefined behavior ([#52999])
```

Comment:
I don't think this is quite precise enough. Infinite loops by themselves were fine. The issue was not making forward progress inside infinite loops, which is a bit of a subtle issue.

Reply:
```suggestion
 - Trivial infinite loops (like `while true; end`) are no longer undefined behavior ([#52999])
```